### DISCUSSION TITLE
Create mm2-offset-syncs topic in target cluster instead of source cluster

### DISCUSSION THREAD

**Discussion by srijita-mallick** [strimzi/strimzi-kafka-operator#10142] [Q&A] [answered]
Hi,

How can we configure the creation of mm2-offset-syncs topic in target cluster instead of source cluster?

Thanks,
Srijita

**Comment by MandMaric** [comment, marked as answer]
https://strimzi.io/docs/operators/0.35.0/full/configuring#con-config-mirrormaker2-connectors-str -> offset-syncs.topic.location

**Reply by srijita-mallick** [reply]
Thanks a lot @MandMaric!
